bnb_yaz/bnb-downloader.py

```diff
@@ -165,8 +181,9 @@ def Main():
     util.Info("Downloaded a total of " + str(total_count) + " new record(s).")
 
 
-try:
-    Main()
-except Exception as e:
-    util.SendEmail("BNB Downloader", "An unexpected error occurred: "
-                   + str(e) + "\n\n" + traceback.format_exc(20), priority=1)
+#try:
```

Comment:
Please either re-enable or remove this try/catch block

Reply:
Done